### ISSUE TITLE
Instructions for installing cookiecutter in 16.1 could stand a little clarification

### ISSUE THREAD

**Issue by JoanEliot** [feldroy/django-crash-course#295]
* Book build date (Is the date on page 1): 2020-06-23
* Book format (PDF, Epub or Mobi): epub
* Chapter or Appendix: 15/16
* Section: 15.3-15.4 -> 16.1

## Description 
Sections 15.3/15.4 instruct us to activate the everycheese environment, but after the big transition to Chapter 16 it would be good to state explicitly where the reader should be, if it matters, when installing cookiecutter. Since I'm not familiar with how cookiecutter installations work, I wanted to be told exactly what environment should be active, if any, and in which folder I should be, if it matters, before running the install command. 

The question I found myself stumbling on is whether (1) I am going to be reinstalling cookiecutter separately in every conda, in which case the everycheese environment should at this point still be active; or (2) I am installing it for reuse by any existing or future conda. But if (2), do I need to be in the 'base' environment or can I run it after exiting out of all environments including base? Likewise, does it matter which folder I'm in? All of this is down to being brand-new to conda, and I understand that you don't want to spend a lot of time explaining infrastructure, but I thought you diverged here from the very tight and comforting hand-holding you'd been doing to that point.

Likewise, 16.3.1 tells me to navigate to the correct project folder, but doesn't say whether or not to activate which environment before running cookiecutter in 16.3.2. I got an error message running it from within the everycheese environment, which I had not been told to deactivate. It worked when running from base, but I didn't try running it with no conda active.

[Off topic: a request for an alternative to GitHub "issues" as the way to identify problems and make suggestions--it's too unstructured. I did try to search for another comment like mine and scrolled through the issue list, but may well have missed such. It would be a lot better for you and for us to be able to go directly to issues organized by chapter/section. Well, maybe this suggestion has also already been made; there's of course a need for a 'general' section as well.]

## Your full name so we can provide accurate credit within the book
Joan Eliot

**Comment by pydanny** [comment]
Thank you for another very observant issue. If you ever want to become an early tech reviewer of one of our future works, I would like to talk to you.

We have address the primary issue, which is where to install Cookiecutter. 

As for the second issue, we could add more labels issues. However, as a company of parents, we are always short on time - we haven't the bandwidth. My apologies.

**Comment by LABranagan** [comment]
I looked through the Django-Crash-Course and then the Cookiecutter documents with 2 questions in mind:  1. Was the installation successful?  2. What would I modify to customize this implementation?

After some study, I determined that most of the files on the GitHub site were copied to my project, perhaps with key substitutions.  But, I could not identify the control file from either set of documentation?

I also saw a large number of files with which I was not familiar.  Which ones are in use in the course?  Which ones are for useful information and recognition?  Which ones are for future (after course) modification?  My suggestion is a graphic or table with the the file name, function / brief description.

I see that cookiecutter appears to have a lot of potential, but I did not find a basic tutorial or explanation of the file structuring that made sense to me.

Thank you
